tests/operators/test_postgres_decorator.py

```diff
@@ -362,10 +362,11 @@ def test_sql_file(self):
 
         cwd = pathlib.Path(__file__).parent
 
+        self.dag.default_args["conn_id"] = "postgres_conn"
+
         with self.dag:
             f = aql.transform_file(
                 sql=str(cwd) + "/test.sql",
-                conn_id="postgres_conn",
```

Comment:
do we still have a test covering the scenario where `conn_id` is defined? It might be worth renaming this test so people can be aware one of its goals is to validate using `transform_file` without `conn_id`